test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,69 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const { expect } = require('chai');
+
+const urlBase = 'https://api.github.com';
+
+describe('Given an authenticate github user', () => {
+  let user;
+
+  before(() => {
+    const request = agent.get(`${urlBase}/user`)
+      .auth('token', process.env.ACCESS_TOKEN)
+      .then((response) => {
+        user = response.body;
+      });
+
+    return request;
+  });
+
+  it('then should have repositories', () => {
+    expect(user.public_repos).to.be.above(0);
+  });
+
+  describe('when get all repositories', () => {
+    let firstRepository;
+
+    before(() => {
+      const request = agent.get(user.repos_url)
+        .auth('token', process.env.ACCESS_TOKEN)
+        .then((response) => {
+          const { body } = response;
+          firstRepository = body.shift();
+        });
+
+      return request;
+    });
+
+    it('then should have some repository', () => {
+      expect(firstRepository).to.not.equal(undefined);
```

Comment:
you depend of a exist repository, you should consider to make this: https://github.com/luigisamurai/psl-workshop-api-testing-js/blob/master/test/RepositoryTest.js#L32